SEED-platform/seed
# feat: add notes modal to inventory list view

#### What's this PR do?
Clicking the _Notes_ icon next to a record from the Inventory List view takes you to the Notes page, but returning to the previous page requires a full reload of all data.

This PR introduces a Notes modal so that all functionality from the Notes page is duplicated in a modal without leaving the Inventory view.

#### How should this be manually tested?
- Verify that all behavior on the Notes page and in the Notes modal works as expected
- Confirm that changing the number of notes in the modal is reflected in the badge next to the record in the Inventory List grid

#### What are the relevant tickets?
#2511 

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/411466/128092828-85aa770a-1ae9-43a8-a119-75fbb205f3bd.png)
